<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="actionShowAboutUsInfo"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Режим</string>
    </property>
    <widget class="QMenu" name="menuText">
     <property name="title">
      <string>Текст</string>
     </property>
     <addaction name="actionSetTextSendMode"/>
     <addaction name="actionSetTextRecieveMode"/>
    </widget>
    <widget class="QMenu" name="menuFile">
     <property name="title">
      <string>Файл</string>
     </property>
     <addaction name="actionSetFileSendMode"/>
     <addaction name="actionSetFileRecieveMode"/>
    </widget>
    <addaction name="menuText"/>
    <addaction name="menuFile"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QMenu" name="menuSwitchPort">
     <property name="title">
      <string>Выбрать порт</string>
     </property>
     <addaction name="actiontest"/>
    </widget>
    <addaction name="actionSwitchColorTheme"/>
    <addaction name="menuSwitchPort"/>
   </widget>
   <addaction name="menuMode"/>
   <addaction name="menuOptions"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSetTextSendMode">
   <property name="text">
    <string>Отправка</string>
   </property>
  </action>
  <action name="actionSetTextRecieveMode">
   <property name="text">
    <string>Приём</string>
   </property>
  </action>
  <action name="actionShowAboutUsInfo">
   <property name="text">
    <string>О нас...</string>
   </property>
  </action>
  <action name="actionSwitchColorTheme">
   <property name="text">
    <string>Сменить цветовую тему</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actionSetFileRecieveMode">
   <property name="text">
    <string>Приём</string>
   </property>
  </action>
  <action name="actionSetFileSendMode">
   <property name="text">
    <string>Отправка</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
